backend/data/src/main/kotlin/io/tolgee/security/ProjectHolder.kt

```diff
@@ -1,9 +1,13 @@
-package io.tolgee.security.project_auth
+package io.tolgee.security
 
 import io.tolgee.dtos.cacheable.ProjectDto
 import io.tolgee.model.Project
 import io.tolgee.service.project.ProjectService
 
+@Deprecated(
```

Comment:
Because it's still used in some places using it to force a project to be in context. I believe these are bad design and should be changed but it was beyond the scope of this PR